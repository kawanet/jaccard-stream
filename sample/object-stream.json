{"item":"item1","user":"user1"}
{"item":"item1","user":"user2"}
{"item":"item2","user":"user2"}
{"item":"item2","user":"user3"}
{"item":"item2","user":"user4"}
{"item":"item3","user":"user1"}
{"item":"item3","user":"user2"}
{"item":"item3","user":"user5"}
