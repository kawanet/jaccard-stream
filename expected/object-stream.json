{"source":"item1","target":"item2","value":0.25}
{"source":"item1","target":"item3","value":0.6666666666666666}
{"source":"item2","target":"item3","value":0.2}
